extensions/eda/plugins/event_source/file_watch.py

```diff
@@ -1,30 +1,39 @@
-r"""file_watch.py.
-
-An ansible-rulebook event source plugin for watching file system changes.
-
-Arguments:
----------
-    path: The directory to watch for changes.
-    ignore_regexes: A list of regular expressions to ignore changes
-    recursive: Recursively watch the path if true
-
-Example:
--------
-    - name: file_watch
-      file_watch:
-        path: "{{src_path}}"
-        recursive: true
-        ignore_regexes: ['.*\\.pytest.*', '.*__pycache__.*', '.*/.git.*']
-
-"""
-
 import asyncio
 import concurrent.futures
 from typing import Any
 
 from watchdog.events import FileSystemEvent, RegexMatchingEventHandler
 from watchdog.observers import Observer
 
+DOCUMENTATION = r"""
+---
+short_description: Load facts from YAML files initially and when the file changes.
```

Comment:
This is incorrect, it does not load fact from yaml facts. that is done by `file` plugin. This plugin watches a directory and sends event for each change inside.